### DISCUSSION TITLE
Works with Android Auto 8?

### DISCUSSION THREAD

**Discussion by lazarevski** [AndreyPavlenko/Fermata#190] [General]
Did anyone test whether Fermata works wit AA8? Thanks. 

(edit: sorry for posting this in 'Issues')

**Comment by olegkr52** [comment]
A little buggy, but it is definitely working.

**Reply by DrUnKeNTiGeR** [reply]
How? I'm on pixel 6 pro AA downgraded to 8.5 on Android 13 beta. I can get fermata control to come up but not fermata. Any suggestions are appreciated.
